### DISCUSSION TITLE
Should we add the bump-brew process to our publish release workflow

### DISCUSSION THREAD

**Discussion by pskrbasu** [Homebrew/discussions#4220] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 3.6.20-143-g10845a1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 10845a1122b916dc91d04f00a4d8b7b888d52654
Last commit: 3 weeks ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 18796e8cf5b476f52054d87073a9982892e2bcbe
Core tap last commit: 3 weeks ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_INSTALL_FROM_API: set
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.39.1 => /opt/homebrew/bin/git
Curl: 7.85.0 => /usr/bin/curl
macOS: 13.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Broken symlinks were found. Remove them with `brew cleanup`:
  /opt/homebrew/opt/postgresql@12
```

### Description of issue

Hi Maintainers,

I just wanted to know if you guys have any workflow currently that determines whether or not we(https://github.com/turbot/steampipe) have a new release ready, and whether you need to update the formula(https://github.com/Homebrew/homebrew-core/blob/master/Formula/steampipe.rb).

The reason for this question is, I was wondering if we could add the bump-brew step to our publish release workflow. If we do add it to our release process, it could create a conflict(or failure in our workflow), if a PR already exists from any of the maintainers(you guys are very quick to raise the PRs ⚡️).

So, should we go ahead and create a step in our workflow to raise PRs ourselves? If yes, then you guys might need to stop creating the PRs(to avoid conflict). Or would you want us to leave this whole process to you guys(since you are the experts here)?

We are good with either of the above options.

Thanks

**Comment by SMillerDev** [comment, marked as answer]
There are two GitHub actions that do this and both will check open pull requests first to check if a new one is needed. In terms of our system to determine updates, we mostly depend on users of Homebrew to send us pull requests.

**Comment by p-linnane** [comment]
This Action is the one we see most often: https://github.com/mislav/bump-homebrew-formula-action
